doc/io/ingestr/index.md

```diff
@@ -44,47 +82,140 @@ Amazon S3, Google Cloud Storage
 Airtable, Asana, GitHub, Google Ads, Google Analytics, Google Sheets, HubSpot,
 Notion, Personio, Salesforce, Slack, Stripe, Zendesk, etc.
 
-## What's inside
+## Configure
+
+For hands-on examples about the configuration parameters enumerated here,
+see [usage section](#usage) below.
+
+:::{rubric} Data source and destination
+:::
+[ingestr] uses four parameters to address source and destination, while
+[`ctk load table`] uses just two, by embedding the source and destination
+table names into the address URLs themselves, using the `table` query
+parameter.
+
+:::{rubric} Batch size
+:::
+Because the underlying framework uses [dlt], you will configure parameters like
+batch size in your `.dlt/config.toml`.
+```toml
+[data_writer]
+buffer_max_items=1_000
+file_max_items=100_000
+file_max_bytes=50_000
+```
 
-The CrateDB destination adapter for ingestr is implemented using [dlt].
+:::{rubric} Incremental loading
+:::
+ingestr supports [incremental loading], which means you can choose to append,
+merge or delete+insert data into the destination table using different
+strategies. Incremental loading allows you to ingest only the new rows from
+the source table into the destination table, which means that you don't have
+to load the entire table every time you run the data migration procedure.
```

Comment:
Note: We will need to evaluate if all strategies work well with CrateDB.

Reply:
CTK also needs to accept and forward ingestr's `--incremental-strategy`, `--incremental-key`, and `--primary-key` CLI options to make it work.